README.md

```diff
@@ -1 +1,164 @@
-# telegraf-cookbook
+# telegraf
+
+Cookbook to install and configure [telegraf](https://github.com/influxdb/telegraf)
+
+This was influenced by [SimpleFinanace/chef-influxdb](https://github.com/SimpleFinance/chef-influxdb)
+
+*Note:* Some plugins will require other packages be installed and that is out of scope for this 
+cookbook.  ie. `[netstat]` requires `lsof`
+
+## Tested Platforms
+
+* CentOS 7.1
+* Ubuntu 14.04
+
+## Requirements
+
+* Chef 12.x
+
+## Usage
+
+This cookbook can be used by including `telegraf::default` in your run list and settings attributes  
+as needed.  Alternatively, you can use the custom resources directly.
+
+### Attributes
+
+<table>
+  <tr>
+    <th>Key</th>
+    <th>Type</th>
+    <th>Description</th>
+    <th>Default</th>
+  </tr>
+  <tr>
+    <td><tt>['telegraf']['version']</tt></td>
+    <td>String</td>
+    <td>Version of telegraf to install, nil = latest</td>
+    <td><tt>nil</tt></td>
+  </tr>
+  <tr>
+    <td><tt>['telegraf']['config_file_path']</tt></td>
+    <td>String</td>
+    <td>Location of the telgraf main config file.</td>
+    <td><tt>'/etc/opt/telegraf/telegraf.conf'</tt></td>
+  </tr>
+  <tr>
+    <td><tt>['telegraf']['config']</tt></td>
+    <td>Hash</td>
+    <td>Hash of config variables to be written to the telegraf config.</td>
+    <td><tt>
+      {
+        'tags' => {},
+        'agent' => {
+          'interval' => '10s',
+          'round_interval' => true,
+          'flush_interval' => '10s',
+          'flush_jitter' => '5s'
+        }
+      }
+    </tt></td>
+  </tr>
+  <tr>
+    <td><tt>['telegraf']['outputs']</tt></td>
+    <td>Hash</td>
+    <td>Hash of telegraf outputs.</td>
+    <td><tt>
+      {
+        'outputs' => [
+          'influxdb' => {
+            'urls' => ['http://localhost:8086'],
+            'database' => 'telegraf',
+            'precision' => 's'
+          }
+        ]
+      }
+    </tt></td>
+  </tr>
+  <tr>
+    <td><tt>['telegraf']['plugins']</tt></td>
+    <td>Hash</td>
+    <td>Hash of telegraf plugins.</td>
+    <td><tt>
+      {
+        'plugins' => {
+          'cpu' => {
+            'percpu' => true,
+            'totalcpu' => true,
+            'drop' => ['cpu_time']
+          },
+          'disk' => {},
+          'io' => {},
+          'mem' => {},
+          'net' => {},
+          'swap' => {},
+          'system' => {}
+        }
+      }
+    </tt></td>
+  </tr>
+</table>
```

Comment:
I think it would be better to use markdown table syntax here vs. inline html. There are online generators that can help you out with formatting.